doc/src/build/wallet.md

```diff
@@ -27,29 +27,51 @@ in [Genesis customization](#customize-genesis).
 1. Optionally, set `RUST_LOG=debug` for verbose logging.
 1. Initiate `genesis`:
    ```shell
-   sui genesis
+   $ sui genesis
    ```
 
+All of this is contained in configuration and keystore files, as well
+as an `authorities_db` database file.
+TODO: Why do I no longer see a `client_db` file created, only `authorities_db` now?
+
 The network configuration is stored in `network.conf` and can be used
 subsequently to start the network. The `wallet.conf` and `wallet.key`
 are also created to be used by the Sui wallet to manage the newly
-created accounts. By default, these files will be stored in the
-`~/.sui/sui_config` directory, but you can override this location by
-providing an alternative path:
+created accounts.
+
+By default, these files are placed in your home directory at
+`~/.sui/sui_config` directory. But you can override this location by
+providing an alternative path to the `--working-dir` argument. Run
+the command like so to place the files in the `dir` directory:
 
 ```shell
-sui genesis --working-dir /path/to/sui/config/files
+$ sui genesis --working-dir /path/to/sui/config/dir
 ```
+
+:note: That path and directory must already exist and will not be
+created with the `--working-dir` argument.
+
 ### Recreating Genesis
 
 To recreate Sui genesis state in the same location, which will remove
-existing configuration files, use the `--force` option to the `sui
-genesis` command.
+existing configuration files, pass the `--force` option to the `sui
+genesis` command and either run it in the same directory or specify
+it once again, using the `--working-dir` argument:
+
+```shell
+$ sui genesis --force --working-dir /path/to/sui/config/dir
+```
+
+For example:
+
+```shell
+$ sui genesis --force --working-dir /Users/name/tmp
```

Comment:
I don't think that this example adds any value on top of the one immediately preceding it.

Reply:
@patrickkuo what do you think?

Reply:
Removing...